plug-ins/clan/command/cclan.cpp

```diff
@@ -141,11 +141,11 @@ void CClan::usage( PCharacter *pc )
         << "{W{lRклан состав{lEclan member{lx{x     показывает лидеру список членов клана (см. {W{lRклан состав помощь{lEclan member help{lx{x)" << endl
         << "{W{lRклан петиция {lEclan petition{lx{x   написать/принять/отклонить петицию на вступление в клан" << endl
         << "                (см. {W{lRклан петиция помощь{lEclan petition help{lx{x)" << endl           
-        << "{W{lRклан дипломатия{lEclan diplomacy {lx{x посмотреть/установить клановую дипломатию (см. {W{lRклан дипломатия помощь{lEclan dipl help{lx{x)" << endl;        
+        << "{W{lRклан дипломатия{lEclan diplomacy {lx{x посмотреть/установить клановую дипломатию (см. {W{lRклан дипломатия помощь{lEclan dipl help{lx{x)" << endl
```

Comment:
As per our discussion, this alone is not enough to re-enable clan status, and it's a separate problem anyway.